src/store/store.js

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import axios from 'axios';
+
+Vue.use(Vuex);
+axios.defaults.baseURL = 'https://exercise-axios.firebaseio.com';
+
+export default new Vuex.Store({
+    state: {
+        movies: [ 
+        ],
+    },
+    getters: {
+        getMovies: state => {
+            return state.movies;
+        },
+    },
+    mutations: {
+        selectMovies: (state) => {
+            //select all movies
+            state.movies = [];
+            axios.get('/movies.json')
+            .then(({data}) => {
+                const newMovies = Object.keys(data).map(item => Object.assign({}, data[item], { id: item }));
+                Vue.set(state, 'movies', [...newMovies]);
+            })
+            .catch(error => console.log(error));
+        },
+    },
+    actions: {
+        selectMovies: ({commit}) => {
```

Comment:
Since we are going to show some movies, the naming convention might be a little confusing. How about 'showMovies' or 'renderMovies' or 'fetchMovies' ? I think that would be more suggestive :)